params/config.go

```diff
@@ -256,12 +256,14 @@ func (c *XDPoSConfig) String() string {
 	return "XDPoS"
 }
 
-/**
-ConsensusVersion will return the consensus version to use for the provided block number. The returned int represent its version
-TODO: It's a dummy value for now until the 2.0 consensus engine is fully implemented.
-*/
-func (c *XDPoSConfig) BlockConsensusVersion(num *big.Int) string {
+func (c *XDPoSConfig) BlockConsensusVersion(num *big.Int, extraByte []byte, skipExtraCheck bool) string {
 	if c.V2 != nil && c.V2.SwitchBlock != nil && num.Cmp(c.V2.SwitchBlock) > 0 {
+		if skipExtraCheck { // first v2 block
```

Comment:
what do you mean by comment "first v2 block"?

Reply:
I shouldn't put this comment here